tools/fetch-team-data.js

```diff
@@ -82,16 +82,20 @@ async function fetchUserProfile(username) {
     }
 
     const { data: profile } = await octokit.users.getByUsername({ username });
+    const { data: social } = await octokit.request('GET /users/{username}/social_accounts', { username });
 
     const result = {
         username: profile.login,
         name: profile.name,
         title: "Guest Author",
-        website: profile.blog,
+        website: profile.blog.match(/http(s)?:\/\//)
+            ? profile.blog
+            : profile.blog ? `https://${profile.blog}` : profile.blog,
```

Comment:
im just wondering why is this condition so complex? 
shouldn't it just be 
```
profile.blog.match(/http(s)?:\/\//) ? profile.blog : `https://${profile.blog}`
```
because profile.blog will always be there in the second condition? 🤔

Reply:
Before I force-push, this is what I wrote. But then I realized that when `profile.blog` was empty, it would become `https://`

https://github.com/eslint/eslint.org/commit/05af50192c5a8df22e4d3baf1931b57ab04d10ec

Reply:
Cool then I had thought blog wouldn't be empty 👍🏻